diff --git a/Makefile b/Makefile
index d9b81f5..e3574a2 100644
--- a/Makefile
+++ b/Makefile
@@ -38,8 +38,6 @@ SERVICES = \
 	locale \
 	modules \
 	net-lo \
-	network \
-	network-pre \
 	pseudofs \
 	random-seed \
 	recovery \
@@ -52,10 +50,8 @@ SERVICES = \
 	tmpfiles-dev \
 	tmpfiles-setup \
 	tmpfs \
-	udevd \
-	udevd-early \
-	udev-settle \
-	udev-trigger
+	mdevd \
+	mdevd-coldplug
 
 TARGETS = \
 	early-console.target \
@@ -175,7 +171,7 @@ install:
 	done
 	# install default service
 	ln -sf ../agetty $(DESTDIR)$(DINITSRVDIR)/boot.d/agetty
-	ln -sf ../udevd $(DESTDIR)$(DINITSRVDIR)/boot.d/udevd
+	ln -sf ../mdevd $(DESTDIR)$(DINITSRVDIR)/boot.d/mdevd
 	# shutdown hook
 	install -Dm755 misc/shutdown-hook $(DESTDIR)$(LIBDIR)/dinit/shutdown-hook
 	# misc
diff --git a/services/binfmt b/services/binfmt
index 117f981..adb806e 100644
--- a/services/binfmt
+++ b/services/binfmt
@@ -1,4 +1,4 @@
 type       = scripted
-command    = /usr/lib/artix/binfmt.sh
+command    = /usr/lib/gentoo/binfmt.sh
 depends-on = early-fs-local.target
 restart    = false
diff --git a/services/cgroups b/services/cgroups
index 941b92b..947473f 100644
--- a/services/cgroups
+++ b/services/cgroups
@@ -1,4 +1,4 @@
 type       = scripted
-command    = /bin/bash /usr/lib/artix/mount-cgroups.sh
+command    = /bin/bash /usr/lib/gentoo/mount-cgroups.sh
 restart    = false
 depends-on = pseudofs
diff --git a/services/early-devices.target b/services/early-devices.target
index b9713c4..be2df02 100644
--- a/services/early-devices.target
+++ b/services/early-devices.target
@@ -1,3 +1,3 @@
 type       = internal
-depends-on = udevd-early
-depends-ms = udev-settle
+depends-on = mdevd
+depends-ms = mdevd-coldplug
diff --git a/services/fsck-root b/services/fsck-root
index 2dafa29..7b3990f 100644
--- a/services/fsck-root
+++ b/services/fsck-root
@@ -4,5 +4,5 @@ restart       = false
 options       = starts-on-console start-interruptible skippable
 start-timeout = 0
 depends-on    = early-prepare.target
-depends-ms    = udevd-early
-after         = udev-settle
+depends-ms    = mdevd
+after         = mdevd-coldplug
